.github/workflows/ci.yml

```diff
@@ -17,7 +17,9 @@ jobs:
               uses: shivammathur/setup-php@v2
               with:
                   php-version: ${{ matrix.php-versions }}
+                  ini-values: xdebug.mode=coverage
                   extensions: intl
+                  tools: composer:v1
```

Comment:
We really need to figure out how to get rid of this because composer v1 is no longer supported

Reply:
Agreed. I was under the impression it was mostly a matter of getting rid of anything < 7.4 and then it would work? Or is it also a matter of updating some dependencies even if they are compatible with 7.4?

Reply:
Composer v2 is perfectly compatible with older versions of PHP. It's mostly related to usage of outdated composer plugins by some dependencies.

Reply:
Is there a way to see what plugins/dependencies are incompatible exactly?

Reply:
try to run a composer install using composer v2, it should tell you which ones are troublesome if I remember correctly

Reply:
https://github.com/cultuurnet/calendar-summary-v3/runs/1727833248?check_suite_focus=true

```
Error: Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - indigophp/doctrine-annotation-autoload v0.1.0 requires composer-plugin-api ^1.0 -> found composer-plugin-api[2.0.0] but it does not match the constraint.
    - cultuurnet/search-v3 dev-master requires indigophp/doctrine-annotation-autoload ^0.1.0 -> satisfiable by indigophp/doctrine-annotation-autoload[v0.1.0].
    - Root composer.json requires cultuurnet/search-v3 dev-master -> satisfiable by cultuurnet/search-v3[dev-master].

You are using Composer 2, which some of your plugins seem to be incompatible with. Make sure you update your plugins or report a plugin-issue to ask them to support Composer 2.

  Problem 1
    - indigophp/doctrine-annotation-autoload v0.1.0 requires composer-plugin-api ^1.0 -> found composer-plugin-api[2.0.0] but it does not match the constraint.
    - cultuurnet/search-v3 dev-master requires indigophp/doctrine-annotation-autoload ^0.1.0 -> satisfiable by indigophp/doctrine-annotation-autoload[v0.1.0].
    - Root composer.json requires cultuurnet/search-v3 dev-master -> satisfiable by cultuurnet/search-v3[dev-master].

You are using Composer 2, which some of your plugins seem to be incompatible with. Make sure you update your plugins or report a plugin-issue to ask them to support Composer 2.
```

Looks like the same problem twice if I'm not mistaken. I'll see if I can get it fixed in `search-v3`